Instrumentation/TocHandler.lua

```diff
@@ -6,10 +6,12 @@ function mod:InjectDependency(lines, dep)
 	local foundDependencyEntry = false
 	local foundPerfyMetadata = false
 	for i, line in ipairs(lines) do
-		local key, value = line:match("^##%s*([^:%s]+)%s*:%s*(.-)%s*$")
+		---@diagnostic disable-next-line: err-esc
+		local _, key, value = line:match("^(\xef?\xbb?\xbf?)##%s*([^:%s]+)%s*:%s*(.-)%s*$")
```

Comment:
the bom itself is unused, can you remove the capture for it?

Reply:
Since lua patterns do not have non-capturing capture groups, the next best thing might be this?
local key, value = select(2, line:match("^(\xef?\xbb?\xbf?)##%s*([^:%s]+)%s*:%s*(.-)%s*$"))